frontend/src/interfaces/ProfileInterface.ts

```diff
@@ -10,8 +10,28 @@ export interface ProfileInterface {
     instagramUrl?: string;
     twitterUrl?: string;
     facebookUrl?: string;
-    following?: string[];
-    followers?: string[];
+    following?: [
+        {
+            following: {
+                profileID: {
+                    profileName: string;
+                    profilePicture: string;
+                    _id: string;
+                };
+            };
+        }
+    ];
```

Comment:
why nesting following inside following? ProfileId is kinf of misleading since this is object not only id.
Also, you may consider using `type: something | null` instead of `type?: something` for more type safety, but that's more general comment I guess

Reply:
👍